packages/react-chat/src/components/Chat/index.tsx

```diff
@@ -63,15 +70,32 @@ const Chat: React.FC<ChatProps> = ({
   const dialogRef = useRef<HTMLElement>(null);
   const [hasAlert, setAlert] = useState(false);
 
+  const { config } = useContext(RuntimeStateAPIContext);
+
+  useEffect(() => {
+    if (autostart) onStart?.();
+  }, [autostart]);
+
   const handleClose = (event: React.MouseEvent<HTMLButtonElement>): void => {
     if (hasEnded) {
       onEnd?.(event);
     } else {
       setAlert(true);
     }
   };
+
   const handleResume = (): void => setAlert(false);
 
+  const actions = useMemo(() => {
+    if (config.render?.mode === RenderMode.BUBBLE) {
+      return [
+        { svg: 'minus', onClick: onMinimize },
+        { svg: 'close', onClick: handleClose },
+      ];
+    }
+    return [{ svg: 'close', onClick: handleClose }];
+  }, [config, handleClose, onMinimize]);
```

Comment:
I don't think we need to memoize this? looks like we weren't before
unless you did this to avoid unnecessary renders in which case 👍